common/djangoapps/django_comment_common/migrations/0003_auto_20161107_0954.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
```

Comment:
Drive-by review comment: Give this file a more descriptive name with the `-n` flag to `makemigrations`